integration/test_archive.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+
+import pytest
+
+from py42.exceptions import Py42ForbiddenError
+
+device_guid = "935873453596901068"
+device_id = "110395"
+org_id = "2689"
+user_id = "176457"
+destination_device_guid = "673679195225718785"
+archive_guid = "912100293346985227"
+path = "C:/Users/QA/Downloads/spatel_add.csv"
```

Comment:
Can we pick a file that does not contain a user's name in it?

Reply:
`spatel` is what I am referring to

Reply:
Shall we discuss this [here](https://github.com/code42/py42/pull/259#discussion_r547085417).

Reply:
all you we need to do is pick another file! There are other files without his name in it

Reply:
changed.